src/service/viewport/viewport-binding-ios-embed-sd.js

```diff
@@ -327,7 +327,10 @@ export class ViewportBindingIosEmbedShadowRoot_ {
   /** @override */
   disableScroll() {
     // TODO(jridgewell): Recursively disable scroll
-    setImportantStyles(this.scroller_, {'overflow-y': 'hidden'});
+    setImportantStyles(this.scroller_, {
+      'overflow-y': 'hidden',
+      'position': 'fixed',
```

Comment:
oh, how does the `position` get reset to whatever it was before? looks like `resetScroll` needs updating as well.

Reply:
I am also worried that setting `position:fixed` on the scroller may have side-effects as it changes the stacking context for all other elements including the "lightbox". I suggest this being behind an experiment flags that's slowly rolled into prod.

Reply:
nevermind, oh realized this is only for the new iOS shadow-dom that @dvoytenko added.